README.md

```diff
@@ -1241,6 +1241,12 @@ These functions can fail, for example if a path does not have an extension, whic
 - `sha256_file(path)` - Return the SHA-256 hash of the file at `path` as a hexadecimal string.
 - `uuid()` - Return a randomly generated UUID.
 
+#### Semantic Versions
+
+- `semver_matches(v, req)` - Check whether a [semantic version][semver] `v` (e.x. `"0.1.0"`) matches a version requirement `req` (e.x. `">=0.1.0"`), returning `"true"` if so and `"false"` otherwise. 
```

Comment:
Rewording a little bit here:

```suggestion
- `semver_matches(version, requirement)`<sup>master</sup> - Check whether a [semantic version](https://semver.org) `version`, e.g., `"0.1.0"` matches requirement `requirement`, e.g., `">=0.1.0"`, returning `"true"` if so and `"false"` otherwise. 
```

I often forget, but I try to put `<sup>VERSION</sup>` in the docs, so people know what version of Just a feature was added, and `<sup>master</sup>` if it's a feature which hasn't made it into a release yet.